### DISCUSSION TITLE
brew services question.

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#2916] [Everyday usage]
Some packages come with files that have the .service extension.  I know that Brew can somehow turn these into launchd plist files.. How is this done. What commands are executed.

There are some other .service files in packages that don't show up when running brew services. Why are these excluded? one of them is gpg-agent.service

How do I go about manually running that service so that it creates and loads a plist.

**Comment by SMillerDev** [comment]
Service files are files for systemd on Linux, brew can't do anything with those on macOS.
